frontend/src/components/Layout/MobileMenu/styled.ts

```diff
@@ -0,0 +1,157 @@
+import styled from 'styled-components';
+import { ReactFragment } from 'react';
+import { Link } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core';
+import { ListItemButton, Backdrop } from '@mui/material';
+
+import theme from 'themeSettings';
+
+import { HEADER_HEIGHT_MOBILE } from '../Header/styled';
+
+interface TabWrapperTypes {
+  component?: ReactFragment;
+  to?: string;
+}
+
+export const MobileMenuSlide = styled('div')({
+  width: '176px',
+  height: `calc(100% - ${HEADER_HEIGHT_MOBILE})`,
+  backgroundColor: theme.palette.secondary.main,
+  padding: '8px 16px 40px 8px',
+  position: 'absolute',
+  top: HEADER_HEIGHT_MOBILE,
+  right: '0px',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'flex-start',
+});
+
+export const MobileMenuBackdrop = styled(Backdrop)({
+  zIndex: '5',
+  background: '#1B1B1C91',
+});
+
+export const MenuTabsWrapper = styled('div')({
+  flexGrow: '0',
+  flexShrink: '0',
+  width: '100%',
+  height: 'fit-content',
+  backgroundColor: theme.palette.secondary.main,
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'flex-start',
+  alignItems: 'left',
+  [theme.breakpoints.up('md')]: {
+    display: 'none',
+  },
+});
+
+export const MenuTabs = styled('div')({
+  fontSize: '24px',
+  fontFamily: '"Ubuntu", sans-serif',
+  margin: '0px',
+});
+
+export const TabWrapper = styled(ListItemButton)<TabWrapperTypes>({
+  height: '50px',
+});
+
+export const SpaceHolder = styled('div')({
+  flexGrow: '2',
+  flexShrink: '1',
+  maxHeight: '100%',
+});
+
+export const MobileUserBlock = styled('div')({
+  flexGrow: '0',
+  flexShrink: '0',
+  height: '32px',
+  display: 'inline-flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  width: '152px',
+  alignItems: 'middle',
+});
+
+export const UserProfile = styled(Link)({
+  display: 'inline-flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  width: '126px',
+  alignItems: 'middle',
+  fontFamily: '"Ubuntu", sans-serif',
+  fontSize: '14px',
+  fontWeight: '400',
+  lineHeight: '16px',
+  textAlign: 'left',
+  color: '#131313',
+});
+
+export const AvatarWrapper = styled('div')({
+  margin: '0px',
+});
+
+export const UserNameWrapper = styled('div')({
+  margin: '0px 10px 0px 8px',
+});
+
+export const UserName = styled('p')({
+  width: '80px',
+  height: '16px',
+  margin: '0px',
+});
+
+export const LogOut = styled('div')({
+  display: 'inline-block',
+  width: '26px',
+  height: '24px',
+  marginTop: '3px',
+  '&:hover': {
+    cursor: 'pointer',
+  },
+});
+
+export const useListStyles = makeStyles({
+  default: {
+    background: 'none',
+    color: 'black',
+    padding: '0px',
+    '& span': {
+      fontFamily: 'Ubuntu, sans-serif',
+      fontWeight: 400,
+      fontSize: '16px',
+      paddingLeft: '2px',
+    },
+    '& svg': {
+      width: '50px',
+      height: '50px',
```

Comment:
I had seen that icons are larger than on mockup. Is actual our version in this file?

Reply:
fixed